soda/core/soda/execution/check/change_over_time_metric_check.py

```diff
@@ -59,6 +59,7 @@ def evaluate(self, metrics: Dict[str, Metric], historic_values: Dict[str, object
             self.historic_diff_values = {
                 "historic_value": historic_value,
                 "metric_value": metric_value,
+                "delta": self.check_value,
```

Comment:
There is an inconsistency in properties you are pushing here compared to the published API of `ChangeOverTimeCoreCheckDiagnostics`. Only `delta` is correct. We have `historicAggregate` and `value`.
But, I think you are pushing this as a `generic` check now, yes? Therefore, all these properties will be lost

Reply:
removing CoT checks from this PR, this will be dealt with separately